scripts/generate_queries_plain.sh

```diff
@@ -0,0 +1,31 @@
+QUERY_TYPES_ALL="\
+cpu-max-all-1 \
+cpu-max-all-8 \
+double-groupby-1 \
+double-groupby-5 \
+double-groupby-all \
+groupby-orderby-limit \
+high-cpu-1 \
+high-cpu-all \
+lastpoint \
+single-groupby-1-1-1 \
+single-groupby-1-1-12 \
+single-groupby-1-8-1 \
+single-groupby-5-1-1 \
+single-groupby-5-1-12 \
+single-groupby-5-8-1"
+
```

Comment:
What about renaming this script to `generate_queries_greptime.sh`? It only generates queries for greptimedb.